### ISSUE TITLE
UI i18n ?

### ISSUE THREAD

**Issue by bzg** [ecosyste-ms/repos#546]
I wonder if it would make sense to allow some i18n for the interface.

Eg https://data.code.gouv.fr says "An open API service providing repository metadata for many open source software ecosystems." which I would like to translate in French since our audience reads French.

I'm willing to contribute if possible - cc'ing @simkim also.

WDYT?

**Comment by andrew** [comment]
Yeah I’d be happy for the UI to be translatable, there can’t be that many strings in there!

**Comment by bzg** [comment]
@simkim is this something you would have time to work on?

**Comment by simkim** [comment]
@bzg Yes !
